### ISSUE TITLE
Better support for multiple player sides

### ISSUE THREAD

**Issue by L-McKendrick** [Soliders-in-Arms-Arma-3-Group/sia3f#57]
Allow the framework to suit multi-sided missions.

- [ ] Per-side briefing settings and configuration.
- [ ] Per-side ACRE settings, including different radio selections and Bable support. ([See MMFW for reference](https://github.com/PiZZAD0X/MMFW/tree/88cae9ba8c6a8c136d901cab401d17e520354bb3/optionals/acre/cfg3DEN/ACRE))
- [ ] Per-side Arsenal options.
- [ ] Per-side roles configuration (is this even possible lmao?)

**Comment by SiegeToaster** [comment]
> Per-side roles configuration (is this even possible lmao?)

yes, it will be the same thing as per-side arsenal options.  idk how to do it yet but I know that it's possible.
